src/app/api/gql/account-info.graphql

```diff
@@ -1,11 +1,7 @@
-mutation AccountInfo ($accessToken: String!) {
+mutation AccountInfo($accessToken: String!) {
```

Comment:
It seems this mutation is not used anywhere in the code, and can be safely removed.

Reply:
I already did this